### PR TITLE
add autotune strength slider and use chromatic array instead of interpolation

### PR DESCRIPTION
**Pull request by Chilluminati91**
When enabling autotune you can now use a slider that changes the strength of the applied effect.

## Description
Interpolate original f0 curve and autotuned f0 curve to set the strength.
Change f0-grid to use an array of chromatic frequencies between 49 - 1046hz.

### CODE REVIEW

**Comment by blaisewf** [comment]
why not a dropdown with all the scales and then a slider with the amount of autotune? like:

imagen

**Comment by Chilluminati91** [comment]
Not too sure whether its needed. Would require one config for each basenote + scale (major, minor, natural minor, harmonic minor, mixo-lydian, etc.) or an intelligent algorithm to calculate them. If you want that kind of flexibility and precision I think you are better off using autotune / wavestune / melodyne etc. in post.

**Comment by AznamirWoW** [comment]
Dunno whether we need this slider. Strength 1 snaps f0 to the nearest note, that's fine.. but 0.5 only goes half-way?

With the new scale I can hear it actually works, but it just flattens the curve

**Comment by Chilluminati91** [comment]
It kinda depends on the skill of the original singer. If you dont have perfect pitch but want to get a slight autotune you can set the slider to 0.3 which tunes the audio enough to make it sound natural but also closer to the intended key. If you're into modern autotune rap you set it to 0.8-1.0 to get the robotic effect.

In the end it just makes the process more adjustable IF you need it, otherwise it is hidden.

**Review by blaisewf — CHANGES_REQUESTED** [review]
UI strings, needs to be added to english i18n, also to TTS tab

**Comment by Chilluminati91** [comment]
added!

File: core.py
Comment (blaisewf) [inline core.py:1]: Missing adding the new value on some parts at the bottom of the CLI

**Review by blaisewf — CHANGES_REQUESTED** [review]

**Review by blaisewf — APPROVED** [review]

**Pull request merged**